tests/tekton-resources/pipelines/eks/awscli-cl2-load-with-addons-slos.yaml

```diff
@@ -96,6 +96,14 @@ spec:
     type: string
   - name: ng-cfn-url-monitoring
     default: "https://raw.githubusercontent.com/awslabs/kubernetes-iteration-toolkit/main/tests/assets/eks_node_group_launch_template.json"
+  - name: dataplane-configuration
+    default: ""
+    type: string
+  - name: eksadm-s3-path
+    default: ""
+    type: string
+  - name: cw-metrics-namespace
```

Comment:
IMO it's OK to have a parameter "is-obelisk" at the pipeline level.
My earlier comment is for Tekton task "CW addon" which is intended to be generic.